<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DbTableEditDialog</class>
 <widget class="QDialog" name="DbTableEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Table Edit</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="windowFilePath">
   <string>Edit tables</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="buttonCreate">
           <property name="text">
            <string>Create table</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonDelete">
           <property name="text">
            <string>Delete Table</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="labelDate">
         <property name="text">
          <string>Creation date :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelComments">
         <property name="text">
          <string>Comments :</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
       <property name="centerButtons">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>DbTableEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>DbTableEditDialog</receiver>
   <slot>onTableSelectionChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCreate</sender>
   <signal>clicked()</signal>
   <receiver>DbTableEditDialog</receiver>
   <slot>onButtonCreate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonDelete</sender>
   <signal>clicked()</signal>
   <receiver>DbTableEditDialog</receiver>
   <slot>onButtonDelete()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onButtonAdd()</slot>
  <slot>onButtonDelete()</slot>
  <slot>onTableSelectionChanged(QString)</slot>
  <slot>onButtonCreate()</slot>
 </slots>
</ui>
